Xamarin.Essentials/Launcher/Launcher.ios.tvos.cs

```diff
@@ -62,9 +65,60 @@ static Task PlatformOpenAsync(OpenFileRequest request)
             return Task.CompletedTask;
         }
 
+        static Task PlatformOpenAsync(OpenFileRequest request, bool openInApp)
+        {
+            if (openInApp)
+            {
+                UIApplication.SharedApplication?.KeyWindow?.RootViewController?.
+                   PresentViewController(
+                       new QLPreviewController() { DataSource = new PreviewDataSource(request.File.FullPath, request.File.FileName) },
+                       true,
+                       null);
+            }
+            else
+            {
+                PlatformOpenAsync(request);
+            }
+            return Task.CompletedTask;
+        }
 #else
         static Task PlatformOpenAsync(OpenFileRequest request) =>
             throw new FeatureNotSupportedException();
+
+        static Task PlatformOpenAsync(OpenFileRequest request, bool openInApp) =>
+            throw new FeatureNotSupportedException();
 #endif
     }
+
+#if __IOS__
+    public class PreviewDataSource : QLPreviewControllerDataSource
+    {
+        readonly string filePath;
+        readonly string fileName;
+
+        public PreviewDataSource(string filePath, string fileName)
+        {
+            this.filePath = filePath;
+            this.fileName = fileName;
+        }
+
+        public override IQLPreviewItem GetPreviewItem(QLPreviewController controller, nint index)
+            => new PreviewItem(filePath, fileName);
+
+        public override nint PreviewItemCount(QLPreviewController controller) => 1;
+    }
+
+    public class PreviewItem : QLPreviewItem
+    {
+        public PreviewItem(string filePath, string fileName)
+        {
+            ItemUrl = NSUrl.FromFilename(filePath);
+            ItemTitle = fileName;
```

Comment:
`fileName` will be used as a window titlel.  Maybe, we should be able to change it.
A file name doesn't always look good